controller/volunteer_controller.py

```diff
@@ -11,6 +11,8 @@ def create_volunteer(username: str,
                      lastname: str,
                      phone: str,
                      camp: Camp) -> Volunteer:
+    if camp.plan.is_closed:
+        raise ControllerError(f"Plan {camp.plan.name} is closed. Please choose another plan.")
```

Comment:
Optional: suggest skipping 'Please choose another plan.', cuz it sounds more like what the interface should handle

Reply:
Done